src/engine/renderer/tr_scene.cpp

```diff
@@ -354,10 +353,9 @@ void RE_AddDynamicLightToScene( const vec3_t org, float radius, float intensity,
 
 	light->additive = true;
 
-	if( light->l.inverseShadows )
-		light->l.scale = -intensity;
-	else
-		light->l.scale = intensity;
+	if ( light->l.inverseShadows ) {
```

Comment:
The forward lighting shader checks the sign of u_LightScale to check for inverse lights. These are said to be already broken, but it would be nice to set the scale to 1 or -1 to at least gesture at how it was supposed to work